src/default/assets/css/_theme.scss

```diff
@@ -0,0 +1,28 @@
+// Default light theme
+:root {
+    --td-color-background: #fdfdfd;
```

Comment:
I'd like to get rid of the `td` prefix. I've never liked the `tsd` prefix we use on all the classes, and once users can easily add custom CSS without creating a custom theme (https://github.com/TypeStrong/typedoc/issues/1060), I want to be able to maintain a pretty stable set of names.

Reply:
Fair enough! I wasn't a big fan of them either, but I put them in to be in line with the `tsd` prefixes. I'll get rid of 'em.

Reply:
Resolved in c29d178dc85abc5db5840b55b5990fed9d7a639c!